test/test_testing.py

```diff
@@ -3337,6 +3337,32 @@ def test_get_pebble_methods(self):
         client = backend.get_pebble('/custom/socket/path')
         self.assertIsInstance(client, _TestingPebbleClient)
 
+    def test_reboot(self):
+        class RebootingCharm(ops.CharmBase):
+            def __init__(self, *args, **kwargs):  # type: ignore
+                super().__init__(*args, **kwargs)  # type: ignore
```

Comment:
I wonder if we should start recommending the more explicit code as follows (also to avoid the `type: ignore`s). What are the trade-offs?

```suggestion
            def __init__(self, framework: ops.Framework):
                super().__init__(framework)
```

Reply:
In some places we do this:

```suggestion
            def __init__(self, *args: Any, **kwargs: Any):
                super().__init__(*args, **kwargs)
```

And that's probably better than the `type: ignore`s I guess.

Since `CharmBase` doesn't take `*args`/`**kwargs` it is indeed probably better to be more explicit. It would only be an issue if the signature changed and lots of updates would be needed then anyway.

Reply:
Ah yeah, I think the `Any`s are fine too, thanks. That's probably better, I guess, as we don't care what the args are here, we just want to forward them.